licenses/templates/licenses/deed_detail.html

```diff
@@ -90,7 +90,7 @@ <h2>
                       <img src="{# {% static '/images/deed/cc_heart_x2.png' %} #}" class="footer-logo" alt="" width="133" height="117" />
                       <div class="footer-trigger-content">
                         <p>{# {% blocktrans %} #}This content is freely available under simple legal terms because of Creative Commons, a non-profit that survives on donations. If you love this content, and love that it's free for everyone, please consider a donation to support our work.{# {% endblocktrans %} #}</p>
-                        <a href="https://us.netdonor.net/page/6650/donate/1?ea.tracking.id=license-footer"><button>Make a Donation</button></a>
+                        <a href="https://creativecommons.org/donate?ea.tracking.id=license-footer"><button>Make a Donation</button></a>
```

Comment:
I hadn't come across the "license-footer" tracking ID for donations yet. It looks like for the Deed Overlay there are maybe two tracking IDs at work?

Assuming what I'm looking at here is the orange slider on Deed pages, and the full script is this (based on page source):

`<noscript>
                  <div id="deed-donate-footer">
                    <div class="footer-trigger">
                      <img src="/images/deed/cc_heart_x2.png" class="footer-logo" alt="" width="133" height="117" />
                      <div class="footer-trigger-content">
                        <p>This content is freely available under simple legal terms because of Creative Commons, a non-profit that survives on donations. If you love this content, and love that it's free for everyone, please consider a donation to support our work.</p>
                        <a href="https://us.netdonor.net/page/6650/donate/1?ea.tracking.id=license-footer"><button>Make a Donation</button></a>
                      </div>
                    </div>
                  </div>
                </noscript>`

Here we need to put a direct Classy link. I haven't found evidence in Classy docs yet to indicate that we can pass through our additional parameters on a redirect like this.

This should therefore be:
https://www.classy.org/give/313412/#!/donation/checkout?c_src=website&c_src2=Deed-Overlay

And not
https://creativecommons.org/donate?ea.tracking.id=license-footer